spec/models/food_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Food, type: :model do
+  describe 'validations' do
```

Comment:
- [ ] Great effort working on your on your project but currently majority of the test cases are not passing. I would kindly suggest you check the test cases you have written and be sure they are working perfectly.

Below is the Screenshot  :point_down: 

<img width="956" alt="7cf8db2d-1047-4af6-9017-e4858d5a5929" src="https://github.com/rahimialijan/recipe-app/assets/78634841/889b50e2-a971-4339-a68b-97872b4f1e6e">